GuillotineMenu/GuillotineMenuViewController.swift

```diff
@@ -89,7 +94,13 @@ class GuillotineMenuViewController: UIViewController {
         }
         
     }
+    
+// MARK: IBAction
 
+    @IBAction func profileButtonTapped(sender: UIButton) {
```

Comment:
@maksTheAwesome could you plz rename this method to something like menuOptionButtonTapped. Thanks.